client/src/fetchers/ratings.js

```diff
@@ -1,6 +1,17 @@
 import axios from 'axios';
 
+const SERVER_BASEURL = 'http://localhost:3000/api/reviews';
+
+
 export default {
   // YOUR REQUESTS HERE
+  getReviews: (id) => {
+    let options = {
+      url: SERVER_BASEURL + `/${id}`,
+      method: 'get',
+    };
+
```

Comment:
This will need to be refactored to make use of the relevant controller and model on the server side. Instead of adding the id to the endpoint, you will need to add a 'param' key with an object containing query items. It should look like this:

```
let options = {
  url: SERVER_BASEURL,
  method: 'get',
  params: { product_id: id }
}
```